Description: set the name of the pkgconfig file as per the debian package liblmdb-dev so that the find_library function call find the .pc file.
Author: Henry-Nicolas Tourneur
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/build.rs
+++ b/build.rs
@@ -10,7 +10,7 @@
     lmdb.push("libraries");
     lmdb.push("liblmdb");
 
-    if !pkg_config::find_library("liblmdb").is_ok() {
+    if !pkg_config::find_library("lmdb").is_ok() {
         cc::Build::new()
                     .file(lmdb.join("mdb.c"))
                     .file(lmdb.join("midl.c"))
